recipes/userver/all/conanfile.py

```diff
@@ -0,0 +1,655 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy, export_conandata_patches, apply_conandata_patches
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.53.0"
+
+class UserverConan(ConanFile):
+    name = 'userver'
+    description = 'The C++ Asynchronous Framework'
+    topics = ('framework', 'coroutines', 'asynchronous')
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://userver.tech/'
+    license = 'Apache-2.0'
+    package_type = "library"
+
+    settings = 'os', 'arch', 'compiler', 'build_type'
+    options = {
+        'shared': [True, False],
+        'fPIC': [True, False],
+        'lto': [True, False],
+        'with_jemalloc': [True, False],
+        'with_mongodb': [True, False],
+        'with_postgresql': [True, False],
+        'with_postgresql_extra': [True, False],
+        'with_redis': [True, False],
+        'with_grpc': [True, False],
+        'with_clickhouse': [True, False],
+        'with_rabbitmq': [True, False],
+        'with_utest': [True, False],
+        'namespace': ['ANY'],
+        'namespace_begin': ['ANY'],
+        'namespace_end': ['ANY'],
+    }
+
+    default_options = {
+        'shared': False,
+        'fPIC': True,
+        'lto': True,
+        'with_jemalloc': False,
+        'with_mongodb': True,
+        'with_postgresql': True,
+        'with_postgresql_extra': False,
+        'with_redis': True,
+        'with_grpc': True,
+        'with_clickhouse': True,
+        'with_rabbitmq': True,
+        'with_utest': True,
+        'namespace': 'userver',
+        'namespace_begin': 'namespace userver {',
+        'namespace_end': '}',
+        'mongo-c-driver/*:with_sasl': 'cyrus',
```

Comment:
As I said, this is not a good fit for current CCI's CI behaviour

Reply:
Ok, I disabled mongo plugin at all. But it would be great to have an opportunity to use sasl here